website/content/docs/commands/acl/binding-rule/create.mdx

```diff
@@ -109,3 +113,25 @@ Modify Time  = 2022-12-20 11:15:22.582568 +0000 UTC
 Create Index = 14
 Modify Index = 14
 ```
+
+Create a new ACL Binding Rule where the selector uses a mathing against a
+single `ClaimMapping` which uses `owner` as its value:
+
+```shell-session
+$ nomad.exe acl binding-rule create \
+    -description "example binding rule" \
+    -auth-method "auth0" \
+    -bind-type "role" \
+    -bind-name "eng-ro" \
+    -selector="`"project-developer`"
+ID           = 698fdad6-dcb3-79dd-dc72-b43374057dea
+Description  = example binding rule
+Auth Method  = github
+Selector     = "value.owner == user"
+Bind Type    = role
+Bind Name    = eng-ro
+Create Time  = 2022-12-20 11:15:22.582568 +0000 UTC
+Modify Time  = 2022-12-20 11:15:22.582568 +0000 UTC
+Create Index = 14
+Modify Index = 14
```

Comment:
This command example and the output don't seem to match in areas such as auth method and selector.

Reply:
yikes. corrected in [c1cf4d5](https://github.com/hashicorp/nomad/pull/17470/commits/c1cf4d5e514bdf863ec362e631e3b023424b8c26)